### DISCUSSION TITLE
Can CMI take over vanilla /locate so we can teleport to the location?

### DISCUSSION THREAD

**Discussion by mrfloris** [Zrips/CMI#4200] [Feature Requests]
`/tp @s 4608 ~ -5968`  <--- default, but i want it ala CMI ---> `/tppos 4608 ~ -5968`
Side question, cmi doesn't support ~ , if /tppos could also support that.. 

It would be nice when we do /locate that when we hover and click for the xyz to teleport, that CMI could perhaps take over, or have an alternative like /cmi locate, that when clicked gives: /tppos x y z , so we can use that command (and so it works).
<img width="870" alt="Screenshot 2020-09-13 at 09 46 06" src="https://user-images.githubusercontent.com/28841349/93013168-345b5b00-f5a6-11ea-9de6-f71dd23d4c1f.png">

**Comment by Zrips** [comment]
i can add better support ~ for tppos command, but locale feature is completely different talk by itself.
